search-resources-api/src/main/java/no/unit/nva/search/IndexPublisher.java

```diff
@@ -0,0 +1,78 @@
+package no.unit.nva.search;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import nva.commons.utils.JacocoGenerated;
+
+import java.net.URI;
+import java.util.Objects;
+
+public class IndexPublisher {
+    private final URI id;
+    private final String name;
+
+    @JacocoGenerated
+    @JsonCreator
+    public IndexPublisher(@JsonProperty("id") URI id,
```

Comment:
As I said in the previous comment, we need to look at the publishers API (I don't think it exists)…I think we have customer API, which is probably the wrong thing…we should have a separate API for publishers (yes, served from the customer table